docs/tutorials/talk-to-kafka.rst

```diff
@@ -0,0 +1,688 @@
+    #. Next to *Access Certificate*, click **Download** and save the ``service.cert`` file.
+    #. Next to *CA Certificate*, click **Download** and save the ``ca.pem`` file.
+
+* If you are going to connect using SASL authentication:
+
+  #. Follow the instructions at `Use SASL Authentication with Apache Kafka® <https://docs.aiven.io/docs/products/kafka/howto/kafka-sasl-auth.html>`_ to enable SASL.
+
+  #. In the *Connection Information* section
+
+     #. Select **SASL** as the **Authentication Method**
+     #. Next to *CA Certificate*, click **Download** and save the ``ca.pem`` file
+     #. Note the *Password* required for the SASL, we'll need it for authentication
+
+* Create the keystore ``client.keystore.p12`` and truststore ``client.truststore.jks`` by following  :doc:`our article on configuring Java SSL to access Kafka <../products/kafka/howto/keystore-truststore>`
+
+.. Warning::
+
+  In the below examples, we just pass the name of the keystore and truststore files, but in actual use, the full path should be used.
+
+You can also use the `Aiven command line tool <https://docs.aiven.io/docs/tools/cli.html>`_ to download the files. See the documentation for `avn service user-creds-download <https://docs.aiven.io/docs/tools/cli/service/user.html#avn-service-user-creds-download>`_
+
+Details on the Aiven for Apache Kafka configuration can be found under the :ref:`kafka-tutorial-reference` section.
+
+.. _tutorial-kaka-python-create-a-topic:
+
+Create a topic
+---------------
+
+A topic in Kafka is a named stream of records that is stored within a Kafka cluster. Let's create a Kafka topic. 
+
+If you have a local Kafka instance running, the command to create the topic might be something like this:
+
+.. code:: bash
+
+    bin/kafka-topics.sh --create --topic demo-topic --bootstrap-server localhost:9092
+
+For an Aiven for Apache Kafka service, you can create the topic from the Aiven console. 
+From the **Topics** tab or the Kafka service overview page, click **Add topic**. Give the topic a name "demo-topic". Click **Add topic**.
+Once this topic is created, we can see that the number of partitions is 1. 
+
+The concept of consumer group and consuming messages on Kafka
+------------------------------------------------------------------
+
+Consumer group is the logical grouping of consumers. In Kafka, the consumer(s) must belong to a consumer group, even if it's the default consumer group. 
+For a Kafka cluster with multiple nodes, consumers within the same consumer group can exist on different nodes. 
+
+.. mermaid::
+
+    graph TD
+
+        A(Topic) -- Partition 3 --> D[/Consumer 3/]
+        A(Topic) -- Partition 4 --> E[/Consumer 4/]
+        subgraph Consumer Group 2
+        D
+        E
+        end
+
+        A(Topic) -- Partition 1 --> B[/Consumer 1/]
+        A(Topic) -- Partition 2 --> C[/Consumer 2/]
+        subgraph Consumer Group 1
+        B
+        C
+        end
```

Comment:
I believe this is wrong, since it seems that is splitting the topic across consumers belonging to two different consumer groups

Reply:
Thanks for the feedback. Fix is via https://github.com/aiven/devportal/pull/1733/commits/4c4cb085be4be48c0e95b7360ff8deec720d9a44.